docs/Backends.md

```diff
@@ -20,15 +20,21 @@ All backends in Glow derive from the [abstract base class
 `Backend`](https://github.com/pytorch/glow/blob/master/include/glow/Backends/Backend.h). There
 are two pure virtual functions all backends must implement:
 
-- `virtual std::unique_ptr<CompiledFunction> compile(Function *F, const Context &ctx) const;`
+- `virtual std::unique_ptr<CompiledFunction> compile(Function *F) const;`
 
-  - This function takes a `Function *F` to compile. `Context &ctx` maps the
-    graph to the concrete execution environment for a specific function. It
-    should return a unique pointer to the
+  - This function takes a `Function *F` to compile. It should return a unique pointer to the
     [`CompiledFunction`](#compiledfunction-abstract-class) of `F`. If the backend
     uses Glow low-level IR, it can call `generateAndOptimizeIR()` to generate an
     optimized `IRFunction`.
 
+- `virtual std::unique_ptr<CompiledFunction> compileWithoutConstants(Function *F) const;`
+
+  - This function takes a `Function *F` to compile. This function does not collect
+    Constants this allows for   that action to be handled later. It should return
```

Comment:
`collect Constants` is it described anywhere? What exactly that is.

Reply:
I added a new section for RuntimeBundle and some more explanation. Should the Bundle go into a different document? It's sorta the bridge between Runtime and Compile time.